vignettes/packageDiscrepencies.Rmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "Package discrepencies"
+author: "T. Joe Mills"
+date: "`r format(Sys.time(), '%d %B, %Y')`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteIndexEntry{Package discrepencies}
+  \usepackage[utf8]{inputenc}
+---
+
+This vignette summarizes discrepancies between the statistic calculations in the new (>v1.0) and the old (<v1.0) EflowStats packages. In general, the authors of the new EflowStats package focused primarily on ensuring that the statistics were calculated according to the definitions stated in Henriksen et al, 2006, rather than on reproducing output from previous package versions or the original HITHAT software. This process uncovered a number of issues with the old EflowStats statistics as well as some statistics calculated in the original HITHAT software (Henriksen et al, 2006). A record of this process can be found in the [Closed issues](https://github.com/USGS-R/EflowStats/issues?q=is%3Aissue+is%3Aclosed) page of the GitHub repository for this package.
+
+## Issues in original HITHAT software
+
+The original HITHAT software contained two bugs in the code that affected a number of the statistics. One bug involved the calculation of monthly means where a single monthly value was returned rather than the mean. This bug affects many statistics that rely on monthly means, such as the MA12-23 statistics. A similar bug was found with 7-day rolling mean calculations and influenced the ML17-18 statistics.
+
+## Old EflowStats package issues
+
+There are a number of discrepancies between the new (v>1.0) and old (v<1.0) EflowStats packages. A list of these by statistic is provided below, but any additional changes to the EflowStats statistics in subsequent versions (>1.0) will be documented in the package _NEWS_ file. 
```

Comment:
Now we just have to remember to use the "NEWS" file. Maybe something to add to the PR template for the package?